amundsen_application/static/js/components/common/ResourceListItem/UserListItem/index.spec.tsx

```diff
@@ -171,56 +171,6 @@ describe('UserListItem', () => {
         expect(resourceType.text()).toEqual('User');
       });
     });
-
-    describe('renders resource-badges section', () => {
-      let resourceBadges;
-      beforeAll(() => {
-        resourceBadges = wrapper.find('.resource-badges');
-      });
-
-      it('renders resource badges section', () => {
-        expect(resourceBadges.exists()).toBe(true);
-      });
-
-      it('does not render Alumni flag if user is active', () => {
```

Comment:
We still need this `<div>` and the `<img>`, just remove the two tests related to the `Flag`